app/views/completed_applications/index.html.erb

```diff
@@ -1,3 +1,95 @@
 <% title t('.page_title') %>
 
-This is a placeholder. TBD.
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full govuk-!-margin-bottom-5">
+    <%= render partial: 'shared/flash_banner' %>
+
+    <h1 class="govuk-heading-xl"><%=t '.heading' %></h1>
+
+    <%= button_to crime_applications_path,
+                  class: 'govuk-button govuk-button--start',
+                  data: { module: 'govuk-button' } do; t('.start_button'); end %>
+  </div>
+</div>
+
+<nav class="moj-sub-navigation" aria-label="Sub navigation">
```

Comment:
I think we may be able to extract the sub navigation into its own partial and just pass as `locals` the different counters?
I've added another comment to make the code simpler too, leveraging the locales, with no need for if-else.